src/algorithms/calorimetry/CalorimeterClusterRecoCoGConfig.h

```diff
@@ -0,0 +1,24 @@
+// Subject to the terms in the LICENSE file found in the top-level directory.
+//
+
+#pragma once
+
+namespace eicrecon {
+
+    struct CalorimeterClusterRecoCoGConfig {
+
+        std::string energyWeight;
+        std::string moduleDimZName;
+
+        double sampFrac = 1.;
```

Comment:
Would it be possible to rename this variable, as is not really the sampling fraction which we necessarily scale with here but rather a more general calibration factor? I think that would improve the readability of the code even further.

Reply:
Not in this PR